internal/services/securitycenter/security_center_automation_resource.go

```diff
@@ -98,9 +92,9 @@ func resourceSecurityCenterAutomation() *pluginsdk.Resource {
 							Type:     pluginsdk.TypeString,
 							Required: true,
 							ValidateFunc: validation.StringInSlice([]string{
-								typeLogicApp,
-								typeLogAnalytics,
-								typeEventHub,
+								string(automations.ActionTypeEventHub),
+								string(automations.ActionTypeLogicApp),
+								string(automations.ActionTypeWorkspace),
```

Comment:
Looking at the API reference it only lists the capitalised values
https://learn.microsoft.com/en-us/azure/templates/microsoft.security/2019-01-01-preview/automations?pivots=deployment-language-arm-template#automationaction-1

To avoid a breaking change would substituting the user input for one of these new values be acceptable? With the aim of changing that in the next major release